### PR TITLE
Create ConversationScopedTest in jsfContainer 2.3 FAT

### LINKED ISSUE #14527: Create ConversationScoped jsfContainer Test
We have unique setup for handing ConversationScoped beans in the com.ibm.jsfContainer bundle, but I do not see any tests for this. We should add at least one to ensure it works properly, especially as we modify the CDI parts of the bundle & packages for EE9.

### PR DESCRIPTION
**Pull request by volosied**
Fixes #14527

ConversationScopedTest is needed to ensure the ConversationScoped Beans works properly when  using the IBMViewHandlerProxy & FacesUrlTransformer classes. 

See them here: https://github.com/OpenLiberty/open-liberty/tree/464735a8588770d8ce055a4505f53c9a029b12ab/dev/com.ibm.ws.jsfContainer/src/com/ibm/ws/jsf/container/cdi

### CODE REVIEW

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsfContainer_fat_2.3

**Comment by volosied** [comment]
#run-libby-bot

File: dev/com.ibm.ws.jsfContainer_fat_2.3/bnd.bnd [hunk 1]
```diff
@@ -25,7 +25,8 @@ src: \
     test-applications/noJsfApp/src,\
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsfContainer_fat_2.3/bnd.bnd:25]: copyright update?

File: dev/com.ibm.ws.jsfContainer_fat_2.3/test-applications/ConversationScopedTest/resources/WEB-INF/web.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsfContainer_fat_2.3/test-applications/ConversationScopedTest/resources/WEB-INF/web.xml:1]: copyright?

File: dev/com.ibm.ws.jsfContainer_fat_2.3/test-applications/ConversationScopedTest/resources/WEB-INF/faces-config.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pnicolucci) [inline dev/com.ibm.ws.jsfContainer_fat_2.3/test-applications/ConversationScopedTest/resources/WEB-INF/faces-config.xml:1]: copyright?

**Comment by volosied** [comment]
#build

**Comment by pnicolucci** [comment]
#run-libby-bot

**Review by pnicolucci — APPROVED** [review]

**Pull request merged**